records/hic_hal/max32620.yaml

```diff
@@ -0,0 +1,18 @@
+common:
+    target:
+        - max32620
+    macros:
```

Comment:
Need to add a core here, for example:
`core:
        - Cortex-M4`

Reply:
Do Cortex-M4 with and w/o FPU have same names in this file?
MAX32620 is based on Arm Cortex-M4 with FPU.

Reply:
Yes it should be M4F

Reply:
Thanks for updating that!